pkg/controller/services.go

```diff
@@ -693,77 +693,90 @@ func (cont *AciController) fabricPathLogger(node string,
 }
 
 func (cont *AciController) opflexDeviceChanged(obj apicapi.ApicObject) {
-	var nodeUpdates []string
 
-	cont.indexMutex.Lock()
-	nodefound := false
-	for node, devices := range cont.nodeOpflexDevice {
-		found := false
+	devType := obj.GetAttrStr("devType")
+	domName := obj.GetAttrStr("domName")
+	ctrlrName := obj.GetAttrStr("ctrlrName")
 
-		if node == obj.GetAttrStr("hostName") {
-			nodefound = true
-		}
+	if ((devType == cont.env.OpFlexDeviceType()) && (domName == cont.config.AciVmmDomain) && (ctrlrName == cont.config.AciVmmController)) {
```

Comment:
nit: Would it help to have a debug statement here - something to the effect of "Processing Opflex device change notification"?

Reply:
Added, thanks!